### DISCUSSION TITLE
Built-in Geofences editieren? Zum Beispiel um bei Ankunft Ladeport zu öffnen…

### DISCUSSION THREAD

**Discussion by albi** [bassmaster187/TeslaLogger#960] [Ideas]
Mir kam heute die Idee meine Stamm-Ionity-Lader zu editieren um dort den Chargeport automatisch zu öffnen und das Ladelimit anzupassen. Ich kann zwar einen neuen Geofence erstellen, aber damit verliere ich doch den eingebauten Geofence, respektive wird mir das dann nicht mehr so schön dargestellt mit den drei Blitzen.

Hab ich was übersehen oder wäre es schön einen bestehenden Geofence um Funktionen erweitern zu können?

**Comment by bassmaster187** [comment]
Admin Panel -> Menü Extras - Geofence - Ionity editieren

**Comment by albi** [comment]
Naja bestehende Geofences lassen sich nicht bearbeiten. Links in der Liste habe ich nur meine selbst angelegten. In der Karte zeigt es einfach den fixen an.

Dachte es wäre schön einen bestehenden um eigene Settings zu erweitern. Dann hat man das nicht doppelt als Geofence.

<img width="928" alt="Bildschirm­foto 2023-02-27 um 13 15 39" src="https://user-images.githubusercontent.com/89451/221561689-f0c8da87-10e6-476d-9b54-b5fa57671805.png">
